index.js

```diff
@@ -44,16 +46,22 @@ module.exports = {
           replacement: "import lodash from 'lodash'\nconst isPlainObject = lodash.isPlainObject"
         }
       ]
-    })
+    });
+
+    reduxTree = esTranspiler(reduxTree, {
```

Comment:
Very cool pro tip - this will simplify a few other shim addons I have that pull this in just the same :)